.github/workflows/ci.yml

```diff
@@ -74,6 +77,9 @@ jobs:
       EARTHLY_INSTALL_ID: "earthly-githubactions"
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v1
+        with:
+          go-version: 1.16
```

Comment:
Hmm... What's this for?

Reply:
The ubuntu-latest runner only has Go 1.15. I need this action in cases where we use go run since the function I used to recurse through .earthly is new to 1.16.

Reply:
Ah right. I had forgotten that we use `go run` in some places.